definitions/ext-jfrog-servers/tests/Log.json

```diff
@@ -0,0 +1,4301 @@
+[
+    {
+      "results": [
```

Comment:
Could you try flattening the JSON structure [here](https://github.com/newrelic/entity-definitions/blob/6fb33e7aa8d4b4f9b3608e463fc1889b9a3a1013/definitions/ext-jfrog-servers/tests/Log.json#L3), so that it's like in this [file](https://github.com/newrelic/entity-definitions/blob/ea5063824c8911029f3755718353110d1e988e16/definitions/ext-pixie_dns/tests/MetricRaw.json#L1) ?